composer.json

```diff
@@ -596,4 +596,4 @@
             ]
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Team, please ignore this file changes, I will be removing this file from this PR since it is not for the FGAC, but for the PR unit test failures.

Reply:
Pls rebase after https://github.com/googleapis/google-cloud-php/pull/5685 is merged. This should go away.